doc/nrf/known_issues.rst

```diff
@@ -1139,6 +1237,26 @@ Assert on writing to flash
 Timeout without sending packet
   A directed advertiser may time out without sending a packet on air.
 
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: HCI Read Local Supported Commands command does not indicate support for HCI LE Set Privacy Mode command.
+  The HCI LE Set Privacy Mode Command is still available.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: An ASSERT occurs when setting advertising data after HCI Reset without setting advertising parameters.
+  In more recent versions of |NCS|, setting advertising data before setting advertising parameters returns an error.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: An ASSERT may occur when writing to flash.
+  For reliable flash operations, please use a more recent version of |NCS|.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: Directed advertisers may time out without sending a packet on air.
+  For reliable behavior in directed advertisers, please use a more recent version of |NCS|.
```

Comment:
This page does not mention issues that are older than v1-0-0. Therefore I would suggest to leave this issue out.